xbmc/input/InputCodingTable.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+/*
+*      Copyright (C) 2005-2013 Team XBMC
+*      http://xbmc.org
+*
+*  This Program is free software; you can redistribute it and/or modify
+*  it under the terms of the GNU General Public License as published by
+*  the Free Software Foundation; either version 2, or (at your option)
+*  any later version.
+*
+*  This Program is distributed in the hope that it will be useful,
+*  but WITHOUT ANY WARRANTY; without even the implied warranty of
+*  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+*  GNU General Public License for more details.
+*
+*  You should have received a copy of the GNU General Public License
+*  along with XBMC; see the file COPYING.  If not, see
+*  <http://www.gnu.org/licenses/>.
+*
+*/
+
+#include <string>
+#include <vector>
+
+#define GUI_CODINGTABLE_LOOKUP_COMPLETED 65000 //pick something high to avoid conflicts with other gui messages
```

Comment:
IMO all "custom" GUI_MSG_FOO defines should be in `GUIUserMessages.h` to avoid any conflicts which are much easier to see if all the definitions are in the same file.
And maybe also rename it to `GUI_MSG_CODINGTABLE_LOOKUP_COMPLETED` to match the naming of the rest.